app/build.gradle

```diff
@@ -141,12 +141,15 @@ android {
     testOptions {
         unitTests {
             returnDefaultValues = true
+            includeAndroidResources = true
         }
     }
 
     viewBinding {
         enabled = true
     }
+    compileSdkVersion 29
+    buildToolsVersion '29.0.3'
```

Comment:
What makes it necessary to specify compileSdkVersion explicitly?  Can we use the latest sdk for compilation?

Reply:
compileSdkVersion needs to be specified. Otherwise the buikd fails with this message:
```
* What went wrong:
A problem occurred configuring project ':app'.
> compileSdkVersion is not specified. Please add it to build.gradle
```

Reply:
Apparently `buildToolsVersion` is not needed. So I removed it.